build

```diff
@@ -66,6 +66,32 @@ replace(path.join(__dirname, '.server-dist'), [
 	{regex: new RegExp(`(require\\\(.*?)(lib)(.*?\\\))`, 'g'), replace: `$1.lib-dist$3`},
 ]);
 
+var databaseExisted = true;
+
+try {
+	require.resolve('./database/sqlite.db');
+} catch (err) {
+	if (err.code !== 'MODULE_NOT_FOUND') throw err; // should never happen
+	databaseExisted = false;
+	console.log('Databse does not exist. Creating one...');
+	fs.writeFileSync(path.resolve(__dirname, 'database/sqlite.db'), '');
+}
+
+var sqlite = require('sqlite');
```

Comment:
``const``

Reply:
we deliberately use es3 in this file because apparently on old versions of node it would otherwise crash instead of printing the 'we require 10+' message

Reply:
In another note, I just realized I was using async/await, which is newer than `.then`, so I refactored it to `.then`.